src/game/Objects/Unit.cpp

```diff
@@ -2900,8 +2912,14 @@ float Unit::MeleeSpellMissChance(Unit *pVictim, WeaponAttackType attType, int32
     // Hit chance depends from victim auras
     if (attType == RANGED_ATTACK)
         hitChance += pVictim->GetTotalAuraModifier(SPELL_AURA_MOD_ATTACKER_RANGED_HIT_CHANCE);
-    else
-        hitChance += pVictim->GetTotalAuraModifier(SPELL_AURA_MOD_ATTACKER_MELEE_HIT_CHANCE);
+    else {
+        // The first hit from gear (and likely talents) was ignored by mobs with > 10 skill difference
+        int32 hitFromAuras = pVictim->GetTotalAuraModifier(SPELL_AURA_MOD_ATTACKER_MELEE_HIT_CHANCE);

```

Comment:
Is there a way to get the total aura modifiers from gear only? The blue post seems to indicate that talents would not be considered in this penalty.

Reply:
Both are auras, so i wouldn't think so.